### DISCUSSION TITLE
Use k3s ctr without sudo

### DISCUSSION THREAD

**Discussion by assafushy** [k3s-io/k3s#7090] [Q&A] [answered]
we are happily using k3s.

and would like to extract an image list form our cluster.

we are using the command:

k3s ctr images ls 

this command requires sudo premmisions.

for automation purposes, could you please add the ability of using it without sudo?

**Comment by brandond** [comment, marked as answer]
Have you tried:
```bash
chmod o+rw /run/k3s/containerd/containerd.sock
```
or
```bash
chgrp MYGROUP /run/k3s/containerd/containerd.sock
```

**Comment by cyprienandres** [comment]
Hello, 
I had the same issue for my automation purpose.

The purpose was to directly add docker images after build to k3s with the command `k3s ctr images import`

Running this command solved my issue ->  `chmod o+rw /run/k3s/containerd/containerd.sock`

Thanks a lot @brandond !
